### PR TITLE
Update geo-steering.md

### PR DESCRIPTION
**Pull request by CFSeiya**
it is related to ticket 2841372. Customer's request

### CODE REVIEW

**Review by cicku — CHANGES_REQUESTED** [review]
Should be _in the same region_

File: content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md [hunk 1]
```diff
@@ -96,4 +96,10 @@ For help finding data center identifiers, refer to [this community thread](https
...
 Any data center not explicitly defined will fall back to using the corresponding `country_pool`, then `region_pool` mapping (if it exists), and finally to associated default pools.
 
-{{<Aside type="note">}}PoP steering is only available to Enterprise customers and only accessible via the API.{{</Aside>}}
\ No newline at end of file
+{{<Aside type="note">}}PoP steering is only available to Enterprise customers and only accessible via the API.{{</Aside>}}
+
+
+### Failover behavior
+
+When there is one pool in the region and it is unavailable, if there is a fallback pool, it would be used.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md]:
```suggestion
A fallback pool will be used if there is only one pool in the region and it is unavailable.
```

File: content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md [hunk 2]
```diff
@@ -96,4 +96,10 @@ For help finding data center identifiers, refer to [this community thread](https
...
+If there are some pools in the region, the order of the pool will be respected. For example, when the 1st pool is down, 2nd pool will be used.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md]:
```suggestion
If there are multiple pools in the same region, the order of the pools will be respected. For example, if the first pool is unavailable, the second pool will be used.
```

**Review by patriciasantaana — APPROVED** [review]

File: content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md [hunk 3]
```diff
@@ -96,4 +96,10 @@ For help finding data center identifiers, refer to [this community thread](https
...
 Any data center not explicitly defined will fall back to using the corresponding `country_pool`, then `region_pool` mapping (if it exists), and finally to associated default pools.
 
-{{<Aside type="note">}}PoP steering is only available to Enterprise customers and only accessible via the API.{{</Aside>}}
\ No newline at end of file
+{{<Aside type="note">}}PoP steering is only available to Enterprise customers and only accessible via the API.{{</Aside>}}
+
+
+### Failover behavior
+
+A fallback pool will be used if there is only one pool in the region and it is unavailable.
```
Comment (patriciasantaana) [inline content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md]:
```suggestion
A fallback pool will be used if there is only one pool in the same region and it is unavailable.
```

**Review by cicku — APPROVED** [review]

**Pull request merged**